Source/Scene/processPbrMaterials.js

```diff
@@ -91,7 +91,16 @@ define([
         var texCoord;
         if (defined(generatedMaterialValues[textureName + 'Offset'])) {
             var textureIndex = generatedMaterialValues[textureName].index;
-            var sampler = gltf.samplers[gltf.textures[textureIndex].sampler];
+            var samplerSelector = gltf.textures[textureIndex].sampler;
+            var sampler;
+            if (!defined(samplerSelector) || !defined(gltf.samplers)) {
+                sampler = {
```

Comment:
From a performance/simplification standpoint, there's no reason to create this object since it's only used to assign repeatS and repeatT below. (Unless I'm missing something).  So this code could be simplified to something like:

```js
var repeatS = 'true';
var repeatT = 'true';

var samplerSelector = gltf.textures[textureIndex].sampler;
if (!defined(samplerSelector) || !defined(gltf.samplers)) {
    var sampler = gltf.samplers[samplerSelector];
    repeatS === WebGLConstants.REPEAT ? 'true' : 'false';
    repeatT === WebGLConstants.REPEAT ? 'true' : 'false';
}
```

This avoid the extra object allocation and saves a couple lines.